code/noRAM/app/src/main/java/com/example/noram/model/Database.java

```diff
@@ -95,6 +102,14 @@ public CollectionReference getEventsRef() {
         return eventsRef;
     }
 
+    /**
+     * Getter method for the photo reference
```

Comment:
"Getter method for the photo reference" - is that accurate? same with below, isn't this a Key reference?

Reply:
github co-pilot at its best I shall fix

Reply:
fixed